### ISSUE TITLE
inputAttrs is not reactive

### ISSUE THREAD

**Issue by mgohin** [ng-select/ng-select#2189]
**Describe the bug**
The `@Input inputAttrs` is not reactive, so we cannot implement on our side things like accessibility.

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):

1. add `[inputAttrs]="myLocalInputAttrs"`
2. you can see after init that my properties are correctly set in html
3. dynamically change `myLocalInputAttrs`
4. html is not updated

**Expected behavior**
Every change to `myLocalInputAttrs` should be reported to html.

**Additional context**
I checked the code and the method `private _setInputAttributes()` is only called in the `ngOnInit`.
https://github.com/ng-select/ng-select/blob/84926f2d1ac0fc5dd6ea99ef9fc309fd108a7add/src/ng-select/lib/ng-select.component.ts#L259

Our current dirty hack is to call from our side this method.

**Comment by mgohin** [comment]
I'd like the issue not to be in stale state :D

**Comment by mauroc8** [comment]
This is a big deal.

**Comment by pavankjadda** [comment]
@mgohin @mauroc8 care to create PR?
